fastapi_backend/app/routes/items.py

```diff
@@ -19,7 +19,7 @@ async def read_item(
 ):
     result = await db.execute(select(Item).filter(Item.user_id == user.id))
     items = result.scalars().all()
-    return [ItemRead.from_orm(item) for item in items]
+    return [ItemRead.model_validate(item) for item in items]
```

Comment:
Why this was changed? Please add comments on your own PRs to explain the reviewer about any non-trivial unrelated changes.

Reply:
It was deprecated, I got the warning and updated it.